migrations/V104__Business_Case_As_Is_Changes.sql

```diff
@@ -0,0 +1,28 @@
+--  Removing 'As Is' from lifecycle_cost_solution enum
+ALTER TYPE lifecycle_cost_solution 
+    RENAME TO lifecycle_cost_solution_temp;
+    
+CREATE TYPE lifecycle_cost_solution AS ENUM ('Preferred', 'A', 'B');
+
+ALTER TABLE estimated_lifecycle_costs
+    RENAME solution TO solution_temp;
+
+ALTER TABLE estimated_lifecycle_costs ADD solution lifecycle_cost_solution not null;
+
+UPDATE estimated_lifecycle_costs SET solution = solution_temp::text::lifecycle_cost_solution;
```

Comment:
If you run this migration with "As Is" solution data already in the table, the rows `WHERE solution_temp = "As is"` are attempted to be set in the `solution` column, but "As Is" isn't a valid type for that enum, so you get an error there.